pallets/credit/src/lib.rs

```diff
@@ -773,17 +836,20 @@ pub mod pallet {
                     cur_era,
                 ));
             }
+            // TODO: for those not continue delegating's account, also need slash credit
```

Comment:
This comment can be removed, because currently we force continue staking for every account.

Reply:
no